src/Core/System/StateMachine/Util/StateMachineGraphvizDumper.php

```diff
@@ -172,7 +172,7 @@ private function addOptions(array $options): string
         $code = [];
         foreach ($options as $k => $v) {
             \assert(\is_string($k));
-            \assert(\is_string($v));
+            \assert($v === null || \is_scalar($v) || $v instanceof \Stringable);
             $code[] = \sprintf('%s="%s"', $k, $v);
```

Comment:
So that `null` values are omitted:
```suggestion
            if ($v !== null) {
                \assert(\is_scalar($v) || $v instanceof \Stringable);
                $code[] = \sprintf('%s="%s"', $k, $v);
            }
```